src/assets/text/homepage.ts

```diff
@@ -2,9 +2,13 @@ export const homepage = {
   appName: "BotBuster Game",
   startNewGame: "Start new game",
   continueGame: "Continue game",
-  openDaily: "Leaderboard",
+  leaderboard: "Leaderboard",
   descriptionPart1: "Who's a bot?, Who's a human?",
   topRankedTitle: "Top 10",
+  aleoSystems: () => {
+    const currentYear = new Date().getFullYear();
```

Comment:
I don't think legally it should always be the current year. Maybe add a TODO to check it properly